api/project.go

```diff
@@ -158,17 +158,30 @@ func (p *appProjectImpl) addImportsInJson(ignoreError bool, imports ...util.Impo
 	json.Unmarshal([]byte(appDescriptorData), &appDescriptor)
 
 	// list existing imports in JSON to avoid duplicates
-	existingImports := make(map[string]bool)
+	existingImports := make(map[string]string)
 	jsonImports, _ := util.ParseImports(appDescriptor.Imports)
 	for _, e := range jsonImports {
-		existingImports[e.CanonicalImport()] = true
+		existingImports[e.ModulePath()] = e.CanonicalImport()
```

Comment:
```e.ModulePath()``` should not be used a key since it will break the alternate syntax and it does not select a unique contrib (the one being updated).
Based on @mellistibco request in #48, we should use method GoImportPath() of ```util.Import``` interface instead since it's the one used to add imports in *imports.go* file and we don't have duplicate in this one, do we?

```suggestion
		existingImports[e.GoImportPath()] = e
```